_commands/basics/uniq.md

```diff
@@ -2,13 +2,13 @@
 ---
 
 uniq
---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+------
+
+`uniq` allows a user to report or filter repeated lines that are adjacent in a file. If no input or output files are given, uniqu will read and write from standard input or output respectfully. 
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+uniq [options] in.txt out.txt
```

Comment:
It's standard to prefix commands with a '$ ' (and a space after the dollar sign) so it's known that the command can be run on the command line.